neutron/assetlist.json

```diff
@@ -94,14 +94,14 @@
           ]
         },
         {
-          "denom": "usdc",
+          "denom": "axlusdc",
           "exponent": 6
         }
       ],
       "type_asset": "ics20",
       "base": "ibc/F082B65C88E4B6D5EF1DB243CDA1D331D002759E938A0F5CD3FFDC5D53B3E349",
       "name": "IBC usdc",
-      "display": "usdc",
+      "display": "axlusdc",
```

Comment:
This doesn't rename the asset. All this property does is inform the number of decimals to show by pointing for one of the denom_units.
To rename the asset to be able to differentiate it from Noble's USDC, I'd recommend updating the `name` and `symbol` to something like "USD Coin (Axelar)" and "axlUSDC", resp.